README.md

```diff
@@ -38,21 +38,28 @@
 
 ## Windows
 1. Install 7-zip and its asar7z plugin.
-2. Open `C:\Users\user_name\AppData\Local\Programs\signal-desktop\resources\app.asar` with 7zip.
-3. Go into stylesheets directory.
-4. Copy the theme you want to use into the directory.
-5. Right click and edit `manifest.css` file and add import statement at the top. `@import "catppuccin-<flavour>.css";` Replace `<flavour>` with flavour you want to use.
-6. Save and close your editor.
-7. Enjoy!
+1. Open `C:\Users\user_name\AppData\Local\Programs\signal-desktop\resources\app.asar` with 7zip.
+1. Go into stylesheets directory.
+1. Copy the theme you want to use into the directory.
+1. Right click and edit `manifest.css` file and add import statement at the top. `@import "catppuccin-<flavour>.css";` Replace `<flavour>` with flavour you want to use.
+1. Save and close your editor.
+1. Enjoy!
 
 ## Linux
-1. Install npm's asar package globally. `sudo npm install -g @electron/asar`
-2. Extract asar into a temporary directory and change directory. `asar e /usr/lib/signal-desktop/resources/app.asar temp/ && cd temp/stylesheets`
-3. Copy and paste theme you want to use into current directory.
-4. Add import statement at the top of `manifest.css` file. `@import "catppuccin-<flavour>.css";` Replace `<flavour>` with flavour you want to use.
-5. Pack the asar. `cd /usr/lib/signal-desktop/resouces && asar p temp/ app.asar`
-6. Remove the temp directory.
-7. Enjoy!
+1. Install `@electron/asar` from npm, i.e. with `npm install -g @electron/asar`
+1. Extract asar into a temporary directory and change directory.
+    ```bash
+    TEMP=$(mktemp -d)
+    FLAVOR=mocha
+    asar e /usr/lib/signal-desktop/resources/app.asar ${TEMP}
+
+    cd ${TEMP}/stylesheets
+    curl https://raw.github.com/.../catppuccin-${FLAVOR}.css -O catppuccin-${FLAVOR}.css
+    echo '@import catppuccin-'${FLAVOR}.css >> manifest.css
+
+    cd /usr/lib/signal-desktop/resources && asar p ${TEMP} app.asar
```

Comment:
`/usr/lib/signal-desktop/resouces/app.asar` needs root privileges to write, so the command should be `cd /usr/lib/signal-desktop/resources && sudo asar p ${TEMP} app.asar`